ci-operator/config/konflux-workspaces/workspaces/konflux-workspaces-workspaces-main.yaml

```diff
@@ -0,0 +1,87 @@
+build_root:
+  project_image:
+    dockerfile_path: ci/Dockerfile
+releases:
+  initial:
+    integration:
+      name: "4.14"
+      namespace: ocp
+  latest:
+    integration:
+      include_built_images: true
+      name: "4.14"
+      namespace: ocp
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 250Mi
+tests:
+- as: e2e
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    labels:
+      region: us-east-2
+    owner: rhtap
+    product: ocp
+    timeout: 1h0m0s
+    version: "4.14"
+  run_if_changed: ^.github/workflows/push.yaml$|^server|^operator|^ci
+  steps:
+    test:
+    - as: e2e
+      cli: latest
+      commands: |
+        make -C server kustomize
+        export KUSTOMIZE="$(realpath server/bin/kustomize)"
+        export TAG="pr-${PULL_NUMBER}-${PULL_PULL_SHA:0:8}"
+        export OPERATOR_IMG="quay.io/konflux-workspaces/workspaces-operator:${TAG}"
+        export SERVER_IMG="quay.io/konflux-workspaces/workspaces-server:${TAG}"
+        export NAMESPACE=workspaces-system
+        export USE_INSECURE_TLS=true
+        unset REPO_NAME
+        echo "${TAG}"
+
+        while [[ -z "$(skopeo list-tags docker://quay.io/konflux-workspaces/host-operator | grep ${TAG})" ]]; do
```

Comment:
would it make sense to wrap these `while` with `timeout`s? The pipeline would timeout after 1h, but waiting for the images 1h probably just don't make sense

Reply:
`timeout` doesn't work great with shell builtins.  I could wrap it with a call to `bash -c`, but considering I'm going to want to revisit some other things after this lands (see previous comments from @ascerra), I'm going to punt on this one until then.